### PR TITLE
Feature/media library

### PR DESCRIPTION
**Pull request by pjosh**
- Media library skeleton 
- Masonry grid

*All data is fake.

### CODE REVIEW

File: app/assets/stylesheets/components/_c-card.scss [hunk 1]
```diff
@@ -1,4 +1,15 @@
+%step {
+  content: "";
+  position: absolute;
+  left: 0;
+  right: 0;
+  margin: auto;
+  height: rem(3px);
+  background: $white;
+}
+
```
Comment (dhakelila) [inline app/assets/stylesheets/components/_c-card.scss]: Could we add a class to manage when should these steps show?

**Comment by dhakelila** [comment]
Only one little comment, but really nice work. 
I like to extract small bits of code as mixins when we are reusing it. 

Little detail, can you add the link to "media library" in the header? Check how are we doing with the others and don't forget menu mobile. ;)

File: app/assets/stylesheets/components/_c-card.scss (hunk 1 above)
Comment (pjosh) [inline app/assets/stylesheets/components/_c-card.scss]: @dhakelila yes! when you set the class "-card-steps" at the same level of "c-card" the steps will be appeared

Comment (dhakelila) [inline app/assets/stylesheets/components/_c-card.scss]: Oh, yes! Thank you! I miss-read that! :)

**Pull request merged**